File: ejercicio4T3.cpp
// insertar a la derecha

#include <iostream>
#include <vector>
using namespace std;

int main() {
    vector<int> numeros = {10, 20, 30};
    numeros.push_back(40); // inserta al final

    for (int n : numeros) {
        cout << n << " ";
    }
    return 0;
}

File: ejercicio4T3.java
// insertar a la derecha

import java.util.arraylist;

public class insertar {
    public static void main(string[] args) {
        arraylist<integer> numeros = new arraylist<>();
        numeros.add(10);
        numeros.add(20);
        numeros.add(30);

        numeros.add(40); // inserta al final
        system.out.println(numeros); // [10, 20, 30, 40]
    }
}